### ISSUE TITLE
Provide more detail for DATABASE_URL 

### ISSUE THREAD

**Issue by auipga** [symfonycorp/symfony-cloud-docs#78]
> no configuration is needed

Source: https://symfony.com/doc/master/cloud/services/mysql.html#from-the-application-container

But...
- without `DATABASE_URL` in `.env` build fails
- it took me hours to find out, that the following works:
 `mysql://user@database.internal:3306/main?sslmode=disable&charset=utf8mb4&serverVersion=mariadb-10.4.0`
via `symfony ssh -- echo $DATABASE_...`
- the next day, I tried `DATABASE_URL=` with no value, which - oh magic - worked too :)

I could not find anything about this in the docs, using the search nor using Google.

This could be documented better.

**Comment by auipga** [comment]
Of course, the given configuration after "no configuration" [1] didn't work either ;)

[1] https://github.com/symfony/recipes/blob/master/doctrine/doctrine-bundle/1.6/manifest.json#L14

**Comment by tucksaun** [comment]
There is a caveat with Doctrine: building the application container requires knowing the targeted database platform version at build time.
This is the reason behind the comments here https://github.com/symfony/recipes/blob/c7d81ffbff3e2335532b21c0f71289a5489151c8/doctrine/doctrine-bundle/1.6/manifest.json#L13 and https://github.com/symfony/recipes/blob/c7d81ffbff3e2335532b21c0f71289a5489151c8/doctrine/doctrine-bundle/1.6/config/packages/doctrine.yaml#L5-L7.
 
I guess that in your application you removed the default in .env and didn't configure the `server_version` either.

**Comment by auipga** [comment]
You're right, I didn't specify the `server_version`. But giving `DATABASE_URL=` in `.env.prod` seems to be enough.
My committed `.env.dev` has a full `DATABASE_URL` with all needed parameters including `server_version`, which shouldn't be used for SymfonyCloud with `env=prod`, right? So for `prod` I didn't provide any `server_version` but it does work.

**Comment by auipga** [comment]
> I guess that in your application you removed the default in .env

Yes, the "default in .env" is sth. like `db_user:db_pass:...`, which of course doesn't work.
> and didn't configure the server_version either.

Even if I provide the `server_version` in yaml, the deployment fails with a missing `DATABASE_URL`.

So, `DATABASE_URL` MUST be set with either no value or `mysql://user@database.internal:3306/main?sslmode=disable&charset=utf8mb4&serverVersion=mariadb-10.4.0` but **both isn't documented**. Or did I miss it?
